src/app/services/game-state-pipes/count-all-votes.pipe.ts

```diff
@@ -0,0 +1,21 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { Observable, map } from 'rxjs';
+import { VoteEntry } from 'src/app/components/game/interface/menu-bar/vote-system/helpers/VoteEntry';
+import { ArraySchema } from '@colyseus/schema';
+
+@Pipe({
+  name: 'countAllVotes',
+})
+export class CountAllVotesPipe implements PipeTransform {
+  transform(voteList$: Observable<ArraySchema<VoteEntry>>): Observable<number> {
+    return voteList$.pipe(
+      map((voteList: ArraySchema<VoteEntry>) => {
+        let voteCount = 0;
+        voteList.forEach((voteEntry: VoteEntry) => {
+          voteCount += voteEntry.castVotes.length;
+        });
+        return voteCount;
```

Comment:
Hier nochmal genauer schauen aber ich meine das sollte so auch mit einem Reducer funktionieren.

```suggestion
     
        return voteList.reduce((prevVal, nextVal) => {
          prevVal +nextVal.castVotes.length;
        }, 0);
```